packages/config/config/devices/0x0063/templates/config_template.json

```
@@ -0,0 +1,468 @@
+{
+	"factory_default": {
+		"label": "Reset to Factory Default",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Normal operation",
+				"value": 0
+			},
+			{
+				"label": "Reset",
+				"value": 1
+			}
+		]
+	},
+	"load_sensing": {
+		"label": "Load Sensing",
+		"valueSize": 1,
+		"defaultValue": 0,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Enable",
+				"value": 1
+			}
+		]
+	},
+	"switch_mode": {
+		"$import": "~/templates/master_template.json#base_enable_disable",
+		"label": "Switch Mode"
+	},
+	"report_method": {
+		"label": "Change Open/Close Report Method",
+		"valueSize": 1,
+		"defaultValue": 1,
+		"unsigned": true,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Notification",
+				"value": 1
+			},
+			{
+				"label": "Basic Set",
+				"value": 2
+			},
+			{
+				"label": "Basic report",
+				"value": 3
+			}
+		]
+	},
+	"adjust_refresh": {
+		"label": "Adjust Refresh",
+		"description": "Adjusts refresh time from 1-55 seconds",
+		"valueSize": 1,
+		"unit": "seconds",
+		"minValue": 0,
+		"maxValue": 55,
+		"defaultValue": 0,
+		"unsigned": true,
+		"options": [
+			{
+				"label": "Default - 10 seconds",
+				"value": 0
+			},
```

```suggestion
			{
				"label": "10 seconds",
				"value": 0
			},
```
We avoid mentioning "default" in the labels, since UIs add that themselves.